apps/orders/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import mixins
+from rest_framework.viewsets import GenericViewSet
+
+from .models import Order, OrderItem
+from .serializers import OrderItemSerializer, OrderSerializer
+
+
+class Orders(
+    mixins.UpdateModelMixin,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    GenericViewSet,
+):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+
```

Comment:
Same thing here... Only admins should see all orders other users should only see their orders and only their orders